animation_nodes/nodes/number/number_range.py

```diff
@@ -28,6 +29,8 @@ class NumberRangeNode(bpy.types.Node, AnimationNode):
     floatStepType: EnumProperty(name = "Float Step Type", default = "START_STEP",
         items = floatStepTypeItems, update = AnimationNode.refresh)
 
+    endPoint: BoolProperty(name = "End Point", default = True, update = executionCodeChanged)
```

Comment:
Bool properties be named like things which can be Yes or No. In this case `useEndPoint` or `includeEndPoint` might be better.

Reply:
Originally named it after numpy's option. But that makes more sense.